book/fault_proofs/deploy.md

```diff
@@ -119,22 +119,24 @@ cargo run --bin vkey --release
    forge script script/fp/DeployOPSuccinctFDG.s.sol --broadcast --rpc-url <RPC_URL> --private-key <PRIVATE_KEY>
    ```
 
-## Contract Parameters
+## Optional Environment Variables
 
 The deployment script deploys the contract with the following parameters:
 
-- **Initial Bond**: 0.01 ETH by default (configurable via `INITIAL_BOND` in wei, so 10000000000000000 wei for 0.01 ETH).
-- **Proof Reward**: 0.01 ETH by default (configurable via `PROOF_REWARD` in wei, so 10000000000000000 wei for 0.01 ETH).
-- **Starting Anchor Root**: Genesis configuration with block number 0.
-- **Access Control**: Permissionless (address(0) can propose and challenge).
+| Variable | Description | Example |
+|----------|-------------|---------|
+| `INITIAL_BOND` | Initial bond for the game. | 1000000000000000 (for 0.001 ETH) |
```

Comment:
```suggestion
| `INITIAL_BOND_ETH` | Initial bond in ETH for the game. | Default: 0.001 ETH |
```

User should supply in ETH, they're less likely to make mistakes that way. Alternatively, you can do it in GWei.